AUTH.md

```diff
@@ -16,7 +16,34 @@ Whenever a request is run towards a WITSML server, the backend will decrypt the
 This is how the flow is when a user has selected a server and will need to authenticate against it. After this is done, a fresh list of wells is fetched.  
 
 
-<img src="./credentials-flow.svg">
+```mermaid
+sequenceDiagram
+    Note left of Frontend: User Select  WITSML <br> server for the first <br> time in current session
```

Comment:
Should it say "User selects" instead of "User Select"?

Reply:
Fixed